### DISCUSSION TITLE
Redirect after password change

### DISCUSSION THREAD

**Discussion by a1730** [authelia/authelia#6581] [Q&A]
First, thank you very much for this awesome project. Authelia is working great for us.

However, we have a peculiar situation in that users are not directed to default_redirection_url after a successful password change flow. The user is shown the Authelia login page instead. The funny thing is that they are redirected properly after a successful login into Authelia. 

What settings knob are we missing? We tried the global default_redirection_url setting but it gives an error and Authelia bails with:
> option 'cookies' must be configured with the per cookie option 'default_redirection_url' but the global one is configured which is not supported"

The application works very well if we remove the global default_redirection_url  except for the aforementioned glitch. Also, Authelia works for other flows except for this redirect issue.

We are on Authelia untagged-v4.37.5 (v4.38.0-beta3, 5d1fe75).

Thank you.

**Comment by mikew** [comment]
That's now in `session.cookies[].default_redirection_url`: https://github.com/authelia/authelia/blob/cb9d0d46c62c214c4a9a912d1992f1c5fb99290a/docs/content/en/configuration/session/introduction.md?plain=1#L36

**Comment by james-d-elliott** [comment]
Sorry for not getting back sooner. I plan to tackle this at some stage it's just not a high priority yet.

**Reply by a1730** [reply]
I'm not on the blocked list. Yay!!!
I'm sorry about the noise regarding UserInfo, user groups...
There is no urgency any more for us, we wrote a UI that leverages the API. 
It is not optimal because we ideally do NOT want to touch user authn (read password) at all but Life is good (for now).
Thank you for this awesome project. It has replaced Auth0 in our small shop 😄
